src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -105,9 +105,27 @@ class WorkspaceReimburseView extends React.Component {
         });
     }
 
+    static getDerivedStateFromProps(props, state) {
+        const distanceCustomUnit = _.find(lodashGet(props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        const unitValue = lodashGet(distanceCustomUnit, 'attributes.unit', 'mi');
+
+        if (!state.unitValue || (unitValue !== state.unitValue)) {
```

Comment:
This `!state.unitValue` doesn't look right. How can `state.unitValue` not have a value? I don't think this is run before constructor but correct me if i'm mistaken.

Reply:
Mm yeah good callout. I was being overly cautious but yeah thinking about it further it isn't needed, as `getDerivedStateFromProps` runs before the `render` method, and we've got a fallback value for `state.unitValue`. I'll create a quick PR to remove it